Basic-Car-Maintenance/Shared/Localizable.xcstrings

```diff
@@ -43,29 +67,48 @@
     },
     "Add Vehicle" : {
       "localizations" : {
-        "de" : {
+        "cs" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Fahrzeug hinzufügen"
+            "value" : "Přidat vozidlo"
           }
         }
       }
     },
     "Add your first maintenance" : {
-
+      "localizations" : {
+        "cs" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Přidat první servisní záznam"
+          }
+        }
+      }
     },
     "Additional Notes" : {
       "localizations" : {
+        "cs" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Další poznámky"
+          }
+        },
         "de" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Weitere Notizen"
+            "value" : "Fahrzeug hinzufügen"
           }
```

Comment:
Is this overwriting the German translation?

Reply:
Oh, it seems like I must’ve missed a line! Will fix that tomorrow. Sorry!

Reply:
Fixed.